install/linux/usr/share/odemis/sim/sparc2-polarizer-sim.odm.yaml

```diff
@@ -0,0 +1,353 @@
+}
+
+# Control the slit position to either fully-open or small (dependent on the spectrometer slit-in)
+"Slit": {
+    class: actuator.FixedPositionsActuator,
+    role: "slit-in-big",
+    children: {"x": "Optical Actuators"},
+    init: {
+        axis_name: "slit",
+        positions: {
+            0: "on", # fully opened
+            -0.0012: "off", # opening based on the small slit
+        },
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+# Controller for moving the 'Redux' stage
+# Note: the S axis can _only_ be moved when the l axis is near the active/engaged position
+"Mirror Actuators": {
+    class: tmcm.TMCLController,
+    role: "mirror",
+    init: {
+        port: "/dev/fake6",
+        address: null,
+        axes: ["s", "l"],
+        ustepsize: [1.e-6, 1.e-6], # m/µstep (big, for the simulator to move fast)
+        refproc: "Standard",
+        refswitch: {"s": 0, "l": 0},
+    },
+    metadata: {
+        # Default position of the mirror engaged (will be improved during alignment)
+        FAV_POS_ACTIVE: {"l": 70.e-3, "s": 5.e-3},  # m, m
+    },
+}
+
+# Internal component to convert between the referential of the mirror actuators
+# and the X/Y referential of the SEM. Used by the mirror alignment functions.
+"Mirror Actuators in XY referential": {
+    class: actuator.ConvertStage,
+    role: "mirror-xy",
+    children: {"orig": "Mirror Actuators"},
+    init: {
+        axes: ["l", "s"], # mapped respectively to X, Y
+        rotation: -1.0471975511965976, # rad (= 60°)
+        # Note: if you want to use absolute moves, add a "translation" too
+    },
+}
+
+"Polarization Analyzer": {
+    class: actuator.CombinedFixedPositionActuator,
+    role: "polarization-analyzer",   # linear
+    children: {"Pol1": "Linear Polarizer", "Pol2": "Quarter Wave Plate"}, # alphabetically sorted
+    init: {
+        axis_name: "pol",
+        caxes_map: ["rz", "rz"], # ["rz", "rz"] # ["linearPol", "qwpPol"]
+        atol: [3.392e-5, 3.392e-5], # rad (~the step size of the motor)
+        fallback: "unspecified position",
+        # It supports 7 positions
+        # position names are hardcoded in odemis, so do not change!
+        positions: {
+            # pos -> pos (linear), pos (qwp)
+            "0pirad": [0.5, 0.0],
+            "qpirad": [0.785398, 0.785398],
+            "hpirad": [1.570796, 1.570796],
+            "3qpirad": [2.356194, 2.356194],
+            "lcirc": [0.785398, 1.570796],
+            "rcirc": [2.356194, 1.570796],
+            "pass-through": [0.0, 1.570796]
+        },
+    },
+    affects: ["Lens1 Mover"],
+}
+
+# Todo change to more generic name pol1
+"Linear Polarizer": {
+    class: actuator.RotationActuator,
+    role: "linear-polarizer", #null, TODO Sabrina change to null
+    children: {"rz": "Optical Actuators"},  # linearPol
+    init: {
+        axis_name: "linear",
+    },
```

Comment:
Note, we might need some extra affects for properly reporting the metadata on the detectors, but let's do that when you add the support for metadata.